src/Tribe/Views/V2/Assets.php

```diff
@@ -437,6 +437,20 @@ public function register() {
 				],
 			]
 		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-events-v2-block-editor-reskin',
+			'tribe-events-v2-block-editor-reskin.css',
```

Comment:
call this `'tribe-events-single-blocks-reskin.css'`

Reply:
I've updated this